cpu/nrf5x_common/radio/Kconfig.nrf5x

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2021 Inria
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
```

Comment:
I think I like the idea of having this shared among nrf CPUs, I was a bit lost with the different files in the beginning , is it necessary that `Kconfig.net` includes `"$(RIOTCPU)/nrf52/radio/Kconfig"` ? And that in turn includes this? Would it work including this one directly?

Reply:
Is this better?

Reply:
oh no bodged something...